src/algorithms/reco/JetReconstruction.cc

```diff
@@ -23,31 +23,32 @@ namespace eicrecon {
     m_log->trace("Initialized");
   }
 
-  std::vector<edm4eic::ReconstructedParticle*> JetReconstruction::execute(
+  edm4eic::ReconstructedParticleCollection* JetReconstruction::execute(
     const std::vector<const edm4hep::LorentzVectorE*> momenta) {
```

Comment:
Would need to use templated type here
```suggestion
    const T &input_collection) {
```